solr/solr-ref-guide/src/scripting-update-processor.adoc

```diff
@@ -0,0 +1,295 @@
+= Scripting Update Processor
```

Comment:
```suggestion
= Script Update Processor
```

And Can we rename this file to remove the "ing"?
The PR shows this file as new; did you just write all this?

Reply:
Will update the name and the file.   I added this file to the Ref Guide, however much (most?) of the content was sourced from the old Solr cwiki.   Part of my goal in this work is to raise the profile of this powerful feature, so I wanted the great content to be visible.  I did manually test all of this stuff (jython, groovy etc) when I first started working on it.

Reply:
Changes made!